MoEPluginInbox/MoEInboxPluginCordinator.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  MoEInboxPluginCordinator.swift
+//  MoEPluginInbox
+//
+//  Created by Rakshitha on 27/06/22.
+//
+
+import Foundation
+
+final class MoEInboxPluginCordinator {
```

Comment:
@RakshithaAcharya There is a typo MoEInboxPluginCordinator -> MoEInboxPluginCoordinator

Reply:
@deepa-iOS-SDK  DONE